package.json

```diff
@@ -22,17 +22,17 @@
     "example": "parcel example.html"
   },
   "peerDependencies": {
-    "react": "^16.7.0-alpha.0"
+    "react": "16.7.0-alpha.0"
```

Comment:
we probably want this to be `>=16.8.0` since users will need to ensure their version includes Hooks but doesn't preclude more recent versions

Reply:
@fouad You're right. Fixed.